packages/fhir-helpers/src/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { isEqual } from 'lodash';
+import { GroupCharacteristic } from '@smile-cdr/fhirts/dist/FHIR-R4/classes/groupCharacteristic';
+import { Coding } from '@smile-cdr/fhirts/dist/FHIR-R4/classes/coding';
+
+/**
+ * finds a characteristic that has the given coding as one of its characteristic.codings
+ *
+ * @param characteristics - group characteristic
+ * @param coding - coding to test for
+ */
+export function getCharacteristicWithCoding(
```

Comment:
Would it be better to use coding `code` to get the characteristics?
Am thinking of a case where I have a code and I don't want to build the whole Coding to do the search.